euth/rates/serializers.py

```diff
@@ -1,15 +1,42 @@
+from django.contrib.contenttypes.models import ContentType
 from rest_framework import serializers
 
+from euth.rates import models as rate_models
+
 from .models import Rate
 
 
 class RateSerializer(serializers.ModelSerializer):
-    user_name = serializers.SerializerMethodField()
+    meta_info = serializers.SerializerMethodField()
 
     class Meta:
         model = Rate
-        read_only_fields = ('id', 'user_name')
+        read_only_fields = ('id', 'meta_info')
         exclude = ('user', 'modified', 'created')
 
-    def get_user_name(self, obj):
-        return str(obj.user.username)
+    def get_meta_info(self, obj):
```

Comment:
@keimlink This serializer contains a lot of database / model related code. Should we move this to somewhere else, or is that just fine? I could see it going either the model oder services.py?